### PR TITLE
Add "stacker" tool to CloudFormation Tips

### PR DESCRIPTION
**Pull request by ThanosBaskous**
cc @phobologic

### CODE REVIEW

**Comment by phobologic** [comment]
How I didn't do this before, I have no clue.  LGTM, thanks @ThanosBaskous .  Do you tend to link to github repos, or docs, because stacker has full docs at http://stacker.readthedocs.io/en/latest/ as well.

File: README.md [hunk 1]
```diff
@@ -1489,6 +1489,7 @@ CloudFormation
 	- Troposphere does not support all of the resources types you can describe with CloudFormation templates.
```
Comment (phobologic) [inline README.md]: I think this is a false assumption due to poor docs in the currently supported section.  That should probably just be removed, it changes so often.  We're usually pretty close to up to date.

Comment (ThanosBaskous) [reply README.md]: 👍 Updated to reflect this.

**Comment by ThanosBaskous** [comment]
👍 @phobologic updated to point to docs instead of repo.

**Comment by phobologic** [comment]
LGTM!

**Pull request merged**